include/gc/Transforms/Passes.td

```diff
@@ -150,6 +150,41 @@ def GpuTilingAndFusion : Pass<"gpu-tiling", "func::FuncOp"> {
            "The maximum workgroup size.">
     ];
 }
+
+def GpuXeVMAttachTarget: Pass<"xevm-attach-target", ""> {
+  let summary = "Attaches a XeVM target attribute to a GPU Module.";
+  let description = [{
+    This pass searches for all GPU Modules in the immediate regions and attaches
+    a XeVM target if the module matches the name specified by the `module` argument.
+
+    Example:
+    ```
+    // File: in.mlir:
+    gpu.module @xevm_module_1 {...}
+    gpu.module @xevm_module_2 {...}
+    gpu.module @xevm_module_1 {...}
+    // mlir-opt --xevm-attach-target="module=xevm.* chip=pvc" in.mlir
+    gpu.module @xevm_module_1 {...}
+    gpu.module @xevm_module_2 {...}
+    gpu.module @xevm_module_1 [#xevm.target<chip = "pvc">] {...}
```

Comment:
This is confusing, the description says it attaches the attribute to all modules matching the name but the example doesn't follow that rule.